ivy/container/experimental/linear_algebra.py

```diff
@@ -171,6 +171,55 @@ def kron(
             out=out,
         )
 
+    @staticmethod
+    def static_matrix_exp(
+        x: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+        /,
+        *,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        return ContainerBase.multi_map_in_static_method(
```

Comment:
So the error happens at this line and it suggests that "multiple_map_in_static_method" does not exits for ContainerBase.

Reply:
check the "static_kron" and "diagflat" just before this function where it uses "cont_multi_map_in_static_method" instead

Reply:
Ohh! I missed that but now it's corrected.